middleware/csrf/session_manager.go

```diff
@@ -27,11 +27,22 @@ func newSessionManager(s *session.Store, k string) *sessionManager {
 
 // get token from session
 func (m *sessionManager) getRaw(c fiber.Ctx, key string, raw []byte) []byte {
-	sess, err := m.session.Get(c)
-	if err != nil {
-		return nil
+	sess := session.FromContext(c)
+	var token Token
+	var ok bool
+
+	if sess != nil {
+		token, ok = sess.Get(m.key).(Token)
+	} else {
+		// Try to get the session from the store
+		storeSess, err := m.session.Get(c)
+		if err != nil {
+			// Handle error
+			return nil
+		}
```

Comment:
sounds not so good. Why do you ignore error?

Reply:
The CSRF middleware lacks a mechanism to handle that error beyond failing the CSRF check. A refactor of the CSRF middleware would be necessary to address this issue.

Reply:
It's okay, can you open this issue for us?

Reply:
@coderabbitai open an issue to improve error handling in the middleware/csrf/session_manager.go and middleware/csrf/storage_manager.go files as discussed here.

Reply:
@sixcolors https://github.com/gofiber/fiber/pull/3021

Reply:
> @sixcolors #3021

I’ll take a look tomorrow